docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -72,17 +76,121 @@ \section{Critical Assumptions}
+		\hline
+		Shop Profile
+		 & Unable to find details of a shop
+		 & Customers will not be able to see the information about a specific shop (e.g., address, phone number, etc.)
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H6-1                                                                                                         \\
+		\hline
+		\multirow{2}{*}{Services}
+		 & Unable to find a service
+		 & Customers and Employees will not be able to see what services are offered by the shop
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H7-1                                                                                                         \\
+		\cline{2-7}
+		~
+		 & Unable to find details of a service
+		 & Customers and Employees will not be able to see the details of a service (e.g., price, estimated time, etc.)
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Refer to H5-1a
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H7-2                                                                                                         \\
+		\hline
+		\multirow{2}{*}{Shop Employees}
+		 & A former employee joins the shop account
+		 & The former employee can view sensitive information and perform unauthorized actions
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item A former employee accepts their invite link to join the shop as an employee after their employment
+			         has been terminated
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Invite links should expire after a set period of time
+			   \item Invite links should only be able to be accepts once
+			   \item Shop owners should be able to revoke access to any employee
+		   \end{enumerate}
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item
+		   \end{enumerate}
+		 & H8-1                                                                                                         \\
+		\cline{2-7}
+		~
+		 & An employee of the shop cannot be found
+		 & Shop owners will not be able view employee's details; Effected employee will not be able to be
+		assigned to work orders or view their profile details
+		 & \begin{enumerate}[label=\alph*., leftmargin=*]
+			   \item Database failure
```

Comment:
could this be due to another failure? e.g., a bug?

Reply:
https://github.com/arkinmodi/project-sayyara/pull/144#discussion_r997616169